fix(proxy-service): randomise schedule start time and fix certificate typo

frankh opened the pull request:
> [!IMPORTANT]
> 👉 _Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review._

## Problem

<!-- Who are we building for, what are their needs, why is this important? -->

<!-- Does this fix an issue? Uncomment the line below with the issue ID to automatically close it when merged -->
<!-- Closes #ISSUE_ID -->

## Changes

<!-- If there are frontend changes, please include screenshots. -->
<!-- If a reference design was involved, include a link to the relevant Figma frame! -->

## Did you write or update any docs for this change?

<!-- Engineers are responsible for doing the first pass at documenting their features and/or code.  -->

- [ ] I've [added or updated the docs](https://posthog.com/handbook/engineering/writing-docs)
- [ ] I've reached out for help from the docs team
- [ ] No docs needed for this change

## How did you test this code?

<!-- Briefly describe the steps you took. -->
<!-- Include automated tests if possible, otherwise describe the manual testing routine. -->

frankh committed:
randomise schedule start time

frankh committed:
fix DNS certificate -> TLS

what is a DNS certificate?

daibhin commented on posthog/temporal/proxy_service/monitor.py:
```diff
@@ -152,7 +152,7 @@ async def check_certificate_status(inputs: CheckActivityInput) -> CheckActivityO
         if response.certificate_status != CertificateState_READY:
             return CheckActivityOutput(
-                errors=["DNS Certificate is not ready"],
+                errors=["TLS Certificate is not ready"],
```
Can't believe I missed this in my thorough review first time around

frankh committed:
move DNS cert not ready to warnings

we'll error if the page cannot be accessed, but the DNS cert not
renewing could be due to using e.g. cloudflare, in which case the proxy
is still active.

Erroring should be broken
